src/frontend/utils/productFilters/getSortedProducts.js

```diff
@@ -0,0 +1,15 @@
+const getSortedProducts = (products, sortBy) => {
+  if (sortBy === "LOW_TO_HIGH")
+    return [...products].sort(
+      (item1, item2) => item1.discountPrice - item2.discountPrice
+    );
+
```

Comment:
why the products array needs to be destructured?

Reply:
sort is not pure function that's why first I made copy of products array